tests/regression/generate_md_table.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+import os.path as osp
+
+import yaml
+from easydict import EasyDict as edict
+
+from mmdeploy.utils import get_backend, get_task_type, load_config
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='from yaml export markdown table')
+    parser.add_argument('yml_file', help='input yml config path')
+    parser.add_argument('output', help='output markdown file path')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    assert osp.exists(args.yml_file), f'File not exists: {args.yml_file}'
+    output_dir, _ = osp.split(args.output)
+    if output_dir:
+        os.makedirs(output_dir, exist_ok=True)
+    backends = [
+        'onnxruntime', 'tensorrt', 'torchscript', 'pplnn', 'openvino', 'ncnn'
```

Comment:
There are more inference backends supported besides those 5, like huawei's CANN, apple's CoreML, Rockchip's RKNN

Reply:
@lvhan028 Personally, too much unsupported models for this kind of backends, which makes the table too large.
Besides, if we really want to include these kind of backends, it's better to do this by those who supported the backends and let them update the regression yam config files.